.github/workflows/pr.yaml

```
@@ -10,13 +10,21 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  pr-builder:
+    needs:
+      #- build
```

```suggestion
      - build
```

I'm guessing this is just leftover from testing, and should this check should be enforced again.

---

Fixed!